charts/azure-service-operator/values.yaml

```diff
@@ -15,7 +15,7 @@ azureClientSecret: ""
 azureUseMI: False
 
 image:
-  repository: mcr.microsoft.com/k8s/azure-service-operator:latest
+  repository: mcr.microsoft.com/k8s/azureserviceoperator:latest
```

Comment:
We can remove this change and keep this PR just for the doc updates. As updating this will break the pipeline. The other PR should fix this.

Reply:
@jananivMS done.